src/claim/definitions.js

```diff
@@ -441,6 +441,135 @@ const definitions = [
     },
     credentialItem: true,
   },
+  {
+    identifier: 'claim-cvc:Type.dateOfAdministration-v1',
+    version: '1',
+    type: 'cvc:Type:date',
+    credentialItem: true,
+  },
+  {
+    identifier: 'claim-cvc:Type.vaccinationName-v1',
+    version: '1',
+    type: 'String',
+    credentialItem: true,
+  },
+  {
+    identifier: 'claim-cvc:Type.Patient-v1',
+    version: '1',
+    type: {
+      properties: [
+        {
+          name: 'name',
+          type: 'claim-cvc:Type.Name-v1',
+        },
+        {
+          name: 'dateOfBirth',
+          type: 'claim-cvc:Identity.dateOfBirth-v1',
+        },
+      ],
+    },
+    credentialItem: true,
+  },
+  {
+    identifier: 'claim-cvc:Type.MedicalCode-v1',
+    version: '1',
+    type: {
+      properties: [
+        {
+          name: 'name',
+          type: 'String',
+        },
+        {
+          name: 'code',
+          type: 'String',
+        },
+        {
+          name: 'codeSystem',
+          type: 'String',
+        },
+        {
+          name: 'codeSystemName',
+          type: 'String',
+        },
+      ],
+    },
+    credentialItem: true,
+  },
+  {
+    identifier: 'claim-cvc:Type.manufacturer-v1',
+    version: '1',
+    type: {
+      properties: [
+        {
+          name: 'name',
+          type: 'String',
+        },
+        {
+          name: 'code',
+          type: 'claim-cvc:Type.MedicalCode-v1',
+        },
+      ],
+    },
+    credentialItem: true,
+  },
+  {
+    identifier: 'claim-cvc:Type.VaccinationRecord-v1',
+    version: '1',
+    type: {
+      properties: [
+        {
+          name: 'patient',
+          type: 'claim-cvc:Type.Patient-v1',
+        },
+        {
+          name: 'dateOfAdministration',
+          type: 'claim-cvc:Type.dateOfAdministration-v1',
+        },
+        {
+          name: 'manufacturer',
+          type: 'claim-cvc:Type.manufacturer-v1',
+        },
+        {
+          name: 'name',
+          type: 'claim-cvc:Type.vaccinationName-v1',
+        },
+        {
+          name: 'detail',
+          type: 'claim-cvc:Type.vaccinationRecordDetail-v1',
+        },
+        {
+          name: 'organisation',
+          type: 'claim-cvc:Type.organization-v1',
+        },
+        {
+          name: 'code',
+          type: 'claim-cvc:Type.MedicalCode-v1',
+        },
+      ],
+      required: ['patient', 'dateOfAdministration', 'name', 'organisation', 'code'],
```

Comment:
For me it makes sense to have those as required. The HumanAPI has them as optional, but I don't know what that means. I think it is fine to have stricter requirements and reject a vaccination if we don't have what we see as required information.